docker-wrapper.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+
+# Variables setting
+HOST=`hostname -i`
+petalscli="/opt/petals-deployer/bin/petals-cli.sh -h $ENV_CLI_HOST -n $ENV_CLI_PORT -u $ENV_CLI_USER -p $ENV_CLI_PASS -y -c "
+
+# Informational dump
+echo "############################ PETALS DEPLOYER ############################"
+echo "###     Starting deploy script with following parameters:" 
+echo " - petals host: $ENV_CLI_HOST" && echo " - petals port: $ENV_CLI_PORT" && echo " - petals user: $ENV_CLI_USER" && echo " - petals pass: $ENV_CLI_PASS"
+echo " - deploy file: $ENV_DEPLOY_FILE"
+echo " - URLS to deploy: $ENV_DEPLOY_URLS"
+echo " - CLI host: ${HOST}"
```

Comment:
Are we sure we get the right IP address?
How about when there are several network interfaces? We write an information that may not be what CLI uses in fact. With respect to the way K8s works... I do not know if CLI works in the right way. It should not be up to CLI to determine what is its location. In K8s, we will use a virtual IP address, and CLI will never be able to determine it. 

@cdeneux Can we force CLI to indicate a user-defined value for its IP address?
Something we could pass through an environment variable?

Reply:
I did it the same way we do in petals `docker-petals-esb-container`, I don't know if it's the best way. I did it because the http server was not able to send files to petals without it (even in `--net=host` mode), I tried setting it as a litteral hostname (like `embedded.http.host="petalsdeployer"`) but it was not working...
I guess what I was missing is setting this litteral in the docker file as well (or maybe something as runtime) to declare it as an alias or something... any ideas ?

Reply:
We need to be able to specify CLI's host as a parameter.
We will discuss it on the chat.